src/components/customForm/useGetCustomFormElements.ts

```diff
@@ -126,6 +126,7 @@ export const useDataSetCustomFormElements = () => {
                     ].concat(),
                     dataSet: dataSetId,
                     paging: false,
+                    order: 'displayName',
```

Comment:
I also noticed the order of api/dataElementOperands was inconsistent, so explicitly adding `order` to make this display in consistent order